src/code/reducers/index.js

```diff
@@ -134,6 +134,14 @@ export default function reducer(state, action) {
         correct: state.correct + 1
       });
     }
+    case actionTypes.ZOOM_CHALLENGE_WON: {
+      let progress = updateProgress(state, true);
```

Comment:
I was thinking that, but left it partly to be the same as the other ones that imply (without actually doing anything) a victory, to make refactoring clearer. Or, at least, that was my vague  half-thought.